modules/mono/doc_classes/GodotSharp.xml

```diff
@@ -11,26 +14,30 @@
 			<return type="void">
 			</return>
 			<description>
-				Attaches the current thread to the mono runtime.
+				Attaches the current thread to the Mono runtime.
 			</description>
 		</method>
 		<method name="detach_thread">
 			<return type="void">
 			</return>
 			<description>
-				Detaches the current thread from the mono runtime.
+				Detaches the current thread from the Mono runtime.
 			</description>
 		</method>
 		<method name="get_domain_id">
 			<return type="int">
 			</return>
 			<description>
+				Returns the engine MonoDomain's ID.
+				[b]Note:[/b] The Mono runtime must be initialized for this method to work (use [method is_runtime_initialized] to check). If the Mono runtime isn't initialized at the time this method is called, the engine will crash.
 			</description>
 		</method>
 		<method name="get_scripts_domain_id">
 			<return type="int">
 			</return>
 			<description>
+				Returns the scripts MonoDomain's ID. On most platforms, this will return a different MonoDomain ID from [method get_domain_id]. However, on non-simulator iOS builds, it will return the same MonoDomain ID as static ahead-of-time compilation is used there due to platform limitations.
```

Comment:
`get_domain_id` should actually return the same as `get_scripts_domain_id` most of the time, unless it's not loaded.